File: analyzers/its/sources/ManuallyAddedNoncompliantIssues.VB/IntentionalFindings/S6588.vb
Imports System

Friend Class S6588
    Private dateTime As Date = New DateTime(1970, 1, 1) ' Noncompliant (S6588)
    Private dateTimeOffset As DateTimeOffset = New DateTimeOffset(1970, 1, 1, 0, 0, 0, TimeSpan.Zero) ' Noncompliant
End Class

File: analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/IntentionalFindings/S6588.cs
using System;

namespace IntentionalFindings
{
    internal class S6588
    {
        private DateTime dateTime = new DateTime(1970, 1, 1); // Noncompliant (S6588)
        private DateTimeOffset dateTimeOffset = new DateTimeOffset(1970, 1, 1, 0, 0, 0, TimeSpan.Zero); // Noncompliant
    }
}
